vpr/src/base/read_place.cpp

```diff
@@ -200,16 +200,36 @@ void read_place_body(std::ifstream& placement_file,
         } else if (tokens[0][0] == '#') {
             continue; //Skip commented lines
 
-        } else if (tokens.size() == 4 || (tokens.size() > 4 && tokens[4][0] == '#')) {
+        } else if ((tokens.size() == 4 || (tokens.size() > 4 && tokens[4][0] == '#')) || (tokens.size() == 5 || (tokens.size() > 5 && tokens[5][0] == '#'))) {
             //Load the block location
             //
-            //We should have 4 tokens of actual data, with an optional 5th (commented) token indicating VPR's
+            // If the place file corresponds to a 3D architecture, it should contain 5 tokens of actual data, with an optional 6th (commented) token indicating VPR's internal block number.
```

Comment:
(commented) token with additional data (VPR's internal block number in the files written by vpr).